content/en/releases/patch-releases.md

```diff
@@ -78,9 +78,9 @@ releases may also occur in between these.
 
 | Monthly Patch Release | Cherry Pick Deadline | Target date |
 | --------------------- | -------------------- | ----------- |
-| March 2024            | 2024-03-08           | 2024-03-13  |
 | April 2024            | 2024-04-12           | 2024-04-17  |
 | May 2024              | 2024-05-10           | 2024-05-15  |
+| June 2024             | 2024-06-07           | 2024-06-11  |
```

Comment:
@sftim do we want to integrate this table into `schedule.yaml`?

Reply:
I'm all for it - feel free to raise an issue.

Reply:
The implementation will be done in https://github.com/kubernetes/release/pull/3547
Parsing can be done right here.